### PR TITLE
#578 #579 refactor the root directory and rename upload directory

### PR DESCRIPTION
**Pull request by igniteeng000**
Refactor the root directory.

### CODE REVIEW

**Comment by abishekvashok** [comment]
Can you make those two lines multi line as highlighted by hound bot @igniteeng000

**Comment by igniteeng000** [comment]
Ok, I will do the changes.

File: v1/app/generate_badges.py [hunk 1]
```diff
@@ -13,7 +13,7 @@ class GenerateBadges:
     def __init__(self):
         self.NUMBER_OF_BADGES_PER_PAGE = 8
         self.APP_ROOT = os.path.dirname(os.path.abspath(__file__))
-        self.UPLOAD_FOLDER = os.path.join(self.APP_ROOT, 'static/uploads')
+        self.UPLOAD_FOLDER = os.path.join(self.APP_ROOT, 'static/badge_backgrounds')
```
[the remark this answers is not in the document]
Comment (igniteeng000) [reply v1/app/generate_badges.py]: fixes this issue in latest commit

**Comment by igniteeng000** [comment]
please take a look at this pr again, i have changed the requested changes

File: v1/app/generate_badges.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (abishekvashok) [reply v1/app/generate_badges.py]: May be @igniteeng000 you can make it multine.

File: backend/app/generate_badges.py [hunk 2]
```diff
@@ -13,7 +13,8 @@ class GenerateBadges:
     def __init__(self):
         self.NUMBER_OF_BADGES_PER_PAGE = 8
         self.APP_ROOT = os.path.dirname(os.path.abspath(__file__))
-        self.UPLOAD_FOLDER = os.path.join(self.APP_ROOT, 'static/uploads')
+        self.UPLOAD_FOLDER = os.path.join(
```
Comment (igniteeng000) [inline backend/app/generate_badges.py]: see this @abishekvashok

File: v1/app/generate_badges.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (igniteeng000) [reply v1/app/generate_badges.py]: i have done that, plz. guide me what i am missing @abishekvashok

File: backend/app/generate_badges.py [hunk 3]
```diff
@@ -13,7 +13,8 @@ class GenerateBadges:
...
+        self.APP_ROOT, 'static/badge_backgrounds')
```
Comment (ParthS007) [inline backend/app/generate_badges.py]: @igniteeng000 Please keep it in one line only

Comment (igniteeng000) [reply backend/app/generate_badges.py]: I can't do this python does not support long lines

**Comment by igniteeng000** [comment]
Closes #578, closes #579

**Comment by abishekvashok** [comment]
@ParthS007 lets merge it this time since he's new. He's done a great job!

**Pull request merged**